data_managers/data_manager_diamond_database_builder/data_manager/data_manager_diamond_database_builder.py

```diff
@@ -12,227 +13,228 @@
             if not chunk:
                 break
-            sys.stderr.write( chunk )
-        sys.exit( return_code )
+            sys.stderr.write(chunk)
+        sys.exit(return_code)
     tmp_stderr.close()
-    os.remove( temp_fasta.name )
-    return dict( value=database_id, name=database_name, db_path="%s.dmnd" % fasta_base_filename )
+    os.remove(temp_fasta.name)
+    return dict(value=database_id, name=database_name, db_path="%s.dmnd" % fasta_base_filename)
 
 
-def _create_symlink( input_filename, target_directory, database_id, database_name ):
+def _create_symlink(input_filename, target_directory, database_id, database_name):
     fasta_base_filename = "%s.fa" % database_id
-    fasta_filename = os.path.join( target_directory, fasta_base_filename )
-    os.symlink( input_filename, fasta_filename )
-    return dict( value=database_id, name=database_name, db_path=fasta_base_filename )
+    fasta_filename = os.path.join(target_directory, fasta_base_filename)
+    os.symlink(input_filename, fasta_filename)
+    return dict(value=database_id, name=database_name, db_path=fasta_base_filename)
 
 
-REFERENCE_SOURCE_TO_DOWNLOAD = dict( ncbi=download_from_ncbi, url=download_from_url, history=download_from_history, directory=copy_from_directory )
+REFERENCE_SOURCE_TO_DOWNLOAD = dict(ncbi=download_from_ncbi, url=download_from_url, history=download_from_history, directory=copy_from_directory)
+
 
 def main():
-    #Parse Command Line
+    # Parse Command Line
     parser = optparse.OptionParser()
-    parser.add_option( '-d', '--dbkey_description', dest='dbkey_description', action='store', type="string", default=None, help='dbkey_description' )
+    parser.add_option('-d', '--dbkey_description', dest='dbkey_description', action='store', type="string", default=None, help='dbkey_description')
     (options, args) = parser.parse_args()
-    
+
     filename = args[0]
-    
-    params = from_json_string( open( filename ).read() )
-    target_directory = params[ 'output_data' ][0]['extra_files_path']
-    os.mkdir( target_directory )
+
+    params = json.loads(open(filename).read())
+    target_directory = params['output_data'][0]['extra_files_path']
+    os.mkdir(target_directory)
     data_manager_dict = {}
 
     database_id = params['param_dict']['database_id']
     database_name = params['param_dict']['database_name']
 
-    #Fetch the FASTA
-    REFERENCE_SOURCE_TO_DOWNLOAD[ params['param_dict']['reference_source']['reference_source_selector'] ]( data_manager_dict, params, target_directory, database_id, database_name )
-    
-    #save info to json file
-    open( filename, 'wb' ).write( to_json_string( data_manager_dict ) )
+    # Fetch the FASTA
+    REFERENCE_SOURCE_TO_DOWNLOAD[params['param_dict']['reference_source']['reference_source_selector']](data_manager_dict, params, target_directory, database_id, database_name)
+
+    # save info to json file
+    open(filename, 'wb').write(json.dumps(data_manager_dict))
```

Comment:
```suggestion
    open(filename, 'w').write(json.dumps(data_manager_dict, sort_keys=True))
```
that'll break the test output but should be more consistent between test runs.